src/styles/blocks/header.scss

```diff
@@ -0,0 +1,81 @@
+.header {
+    display: inline-flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    height: $header-height;
+    align-items: center;
+
+    box-shadow: 0 2 4 0 rgb(0, 0, 0, 0.05);
+
+    &__links {
+        padding-right: 50px;
+        align-items: center;
+        line-height: $header-height;
+    }
+
+    &__links-list {
+        display: inline-flex;
+        list-style: none;
+        margin: 0;
+        padding: 0;
+        max-height: $header-height;
+    }
+
+    &__links-link {
+        color: $link-color-inactive;
```

Comment:
Ensure that the `$link-color-inactive` variable is defined in your SCSS files. If it's not defined, it will cause a compilation error.